R/write.R

```diff
@@ -72,18 +72,29 @@ write_csv <- function(x, path, na = "NA", append = FALSE, col_names = !append) {
 
 #' @rdname write_delim
 #' @export
-write_excel_csv <- function(x, path, na = "NA", append = FALSE, col_names = !append) {
+write_excel_csv <- function(x, path, delim = ",", na = "NA", append = FALSE, col_names = !append) {
   stopifnot(is.data.frame(x))
 
-  x_out <- lapply(x, output_column)
-  stream_delim(x_out, path, ",", col_names = col_names, append = append,
+  x_formated <- if(delim == ";") {
+    dplyr::mutate_if(x, is.numeric, dplyr::funs(format(., decimal.mark = ",")))
+  } else {
+    x
+  }
+  x_out <- lapply(x_formated, output_column)
+  stream_delim(x_out, path, delim, col_names = col_names, append = append,
     na = na, bom = TRUE)
 
   invisible(x)
 }
 
 #' @rdname write_delim
 #' @export
+write_excel_csv2 <- function(x, path,  delim = ";", na = "NA", append = FALSE, col_names = !append) {
```

Comment:
Why did you choose the name `write_excel_csv2`? Will it be helpful to users who have problems with saving csv with utf8 characters, will they be able to find this function to resolve their problems? Can we come up with a more precise and helpful name?

Reply:
I understand this is analogy to `write.csv2`

Reply:
hmm, right :)

I would say `write.csv2` is poorly named ;) but maybe it makes sense to use similar name

Reply:
yes, this is the only reason why it's called like this :) readr already has `read_csv2` what is reading csv from `write.csv2` function.

Reply:
> eadr already has read_csv2
okay i think this is a solid argument :)